vars/buildExecute.groovy

```diff
@@ -87,8 +91,7 @@ void call(Map parameters = [:]) {
             case 'npm':
                 npmExecuteScripts script: script, install: config.npmInstall, runScripts: config.npmRunScripts
                 break
-            case ['docker', 'kaniko']:
-                kanikoExecute script: script
+            case ['docker', 'kaniko']: //handled below
```

Comment:
I would keep this as it was an just skip if config.cnbBuild is active to have the build steps here together as before.

Reply:
docker and cnbBuild can run together, so I think we would need to separate these build tool cases. Do you still want that here?

Reply:
it's fine, we can also leave it like this.